src/components/summary.json

```
@@ -609,5 +609,44 @@
         }
       ]
     }
+  ],
+  "ExternalDataSet": [
+    {
+      "name": "PDMP Prescriptions",
+      "dataKey": "PDMPMedications",
+      "dataKeySource": "ExternalDataSet",
+      "info": [
+        {
+          "type": "elements",
+          "description": "The medication order queries the following elements:",
+          "elements": [
+            {
+              "name": "MedicationOrder",
+              "status": "any",
+              "lookback": "n/a"
+            }
+          ]
+        }
+      ],
+      "tables": [
+        {
+          "headers": {
+            "Drug": {
+              "key": "medicationCodeableConcept",
+              "formatter": "codeableConceptFormat",
+              "formatterArguments": ["medicationCodeableConcept", "text"],
+              "minWidth": 200
+            },
+            "Written": { "key": "dateWritten", "formatter": "datishFormat" },
+            "NIC Code": {
```

I think this should be NDC (National Drug Code); NIC (eGov) is the vendor providing the PDMP service
```suggestion
            "NDC Code": {
```

---

....I guess *NDC code* is redundant then :)

---

thank you Ivan for catching that. should be fixed now.